File: 1533/1533.go
func findTheIndexOfTheLargeInteger(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findTheIndexOfTheLargeInteger(root.Left)
	r := findTheIndexOfTheLargeInteger(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 1533/1533.py
class Solution:
    def find_the_index_of_the_large_integer(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.find_the_index_of_the_large_integer(root.left), self.find_the_index_of_the_large_integer(root.right))